webpack.config.js

```diff
@@ -251,19 +251,8 @@ module.exports = (env = process.env.NODE_ENV || 'development') => {
           ],
         },
         {
-          test: /\.js$/u,
-          include: [
-            matchModule('ansi-styles'),
-            matchModule('ast-types'),
-            matchModule('chalk'),
-            matchModule('lodash-es'),
-            matchModule('parse5'),
-            matchModule('parse5-sax-parser'),
-            matchModule('postcss-html'),
-            matchModule('recast'),
-            matchModule('redux'),
-            matchModule('stylelint'),
-          ],
+          test: /\.[jt]sx?$/u,
```

Comment:
Hmm, if we actually did encounter a TypeScript file (which is not something we would expect in a node module) we wouldn’t be able to compile it, would we? Since we don’t have the babel TS compiler in the build chain?

So it seems misleading to include `.ts` in the extensions this rule catches…

Reply:
You are correct, this implies support for typescript in a way that doesn't make sense.